src/envoy/core.clj

```diff
@@ -37,6 +38,26 @@
            [(if to-keys? (keyword Key) Key)
             (when Value (fromBase64 Value))]))))
 
+(defn- round-robin
```

Comment:
this can be improved to something like:
```clojure
=> (defn find-consul-node [hosts]
     (let [at (atom -1)]
       #(nth hosts (mod (swap! at inc)
                        (count hosts)))))
#'user/find-consul-node
```
```clojure
=> (def consul-node (find-consul-node ["foo.com" "bar.com" "baz.com"]))
#'user/consul-node
```
```clojure
=> (consul-node)
"foo.com"
=> (consul-node)
"bar.com"
=> (consul-node)
"baz.com"
=> (consul-node)
"foo.com"
=> (consul-node)
"bar.com"
```